paloma-testnet-11/genesis.json

```diff
@@ -8217,23 +8217,23 @@
             "@type": "/cosmos.upgrade.v1beta1.SoftwareUpgradeProposal",
             "description": "The Paloma flock of validators need to upgrade Paloma-Testnet-10 with a breaking change. This upgrade delivers fixes for validator apphash failures due to the inability to verify cosmwasm contracts in state sysnc. Paloma wasmd v0.25.0 lacked app.go support for snapshot of cosmwasm contracts for use with state-sync. This upgrade will allow for more permissionless cosmwasm and NFT activity on Paloma chain that will not cause a majority of validators to fail out of the network. In addition, this release provides the first iteration of the protofiles that will be used to allow anyone to schedule messaging jobs on Paloma for the pigeons to relay.",
             "plan": {
-              "height": "562902",
-              "info": "{\"binaries\":{\"linux/amd64\":\"https://github.com/palomachain/paloma/releases/download/v0.10.1/paloma_Linux_x86_64.tar.gz\"}}",
-              "name": "v0.10.1",
+              "height": "532422",
+              "info": "{\"binaries\":{\"linux/amd64\":\"https://github.com/palomachain/paloma/releases/download/v0.10.0/paloma_Linux_x86_64.tar.gz\"}}",
+              "name": "v0.10.0",
```

Comment:
tag and link to binary that has the correct wasmd version that we believe to work